common/attest_plugin.c

```diff
@@ -40,6 +44,50 @@ const char* OE_OPTIONAL_CLAIMS[OE_OPTIONAL_CLAIMS_COUNT] = {
 // Variables storing the verifier list.
 static oe_plugin_list_node_t* verifiers = NULL;
 
+// UUID for all OE reports generated by oe_get_report().
+static const oe_uuid_t _ecdsa_report_uuid = {
+    OE_FORMAT_UUID_SGX_ECDSA_P256_REPORT};
+
+// verify report user data against peer certificate
+static oe_result_t verify_report_user_data(
```

Comment:
Add sgx to function name.

Also this file is nominally a TEE-agnostic file, and should not normally have any SGX-only functions in this file.  Such code should be in a filename with "sgx" in the filename (or directory name)

Reply:
Specifically, I think this function should be down in the SGX plugin, rather than in the core plugin framework